lib/elastic_apm.rb

```diff
@@ -62,6 +62,26 @@ def current_span
       agent&.current_span
     end
 
+    # rubocop:disable Metrics/AbcSize
+    # Get a formatted string containing transaction, span, and trace ids.
+    # If a block is provided, the ids are yielded.
+    #
+    # @yield [String, String, String] The transaction, span, and trace ids.
```

Comment:
Do these need to be declared `String|nil`?

Reply:
I think so? Is that the proper way to notate that?

Reply:
Clearly I don't know: https://github.com/elastic/apm-agent-ruby/blob/master/lib/elastic_apm.rb#L53 😀

Reply:
I have no idea either = )
I've commit this...what do you think? https://github.com/elastic/apm-agent-ruby/pull/527/commits/53b66d11b0e9287c673c0543eaa03efb786302e6